https://issues.redhat.com/browse/ACM-14437 (#7230)
open-cluster-management/rhacm-docs: clusters/cluster_lifecycle/release_image_disconn.adoc

. Add the YAML files for the images that you want available when you create a cluster by using the console or CLI to manually add the `clusterImageSet` YAML content.

> . Modify the `clusterImageSet` YAML files for the remaining {ocp-short} release images to reference the correct offline repository where you store the images. Your updates resemble the following example where `spec.releaseImage` refers to the image registry or digest that you are using:

Author: Do we want to say only digest here?

Reviewer: Hmm I think something more like

```suggestion
. Modify the `clusterImageSet` YAML files for the remaining {ocp-short} release images to reference the correct offline repository where you store the images. Your updates resemble the following example where `spec.releaseImage` uses your offline image registry of the release image. Additionally, the release image should referenced by digest:
```

This field has both:
1. the release image using their offline image registry (aka mirror registry) and
2. be referenced by digest 

Of course, please feel free to modify the wording!

metadata:
  labels:
    channel: fast
  name: img<4.x.x>-x86-64-appsub
spec:
  releaseImage: IMAGE_REGISTRY_IPADDRESS_or__DNSNAME/REPO_PATH/ocp-release@sha256:073a4e46289be25e2a05f5264c8f1d697410db66b960c9ceeddebd1c61e58717
----

. Ensure that the images are loaded in the offline image registry that is referenced in the YAML file. 

> . If you use digest, you need to obtain the image digest by running the following command:

Reviewer: Digest is a requirement, so maybe the wording could be more like an additional instruction

```suggestion
. To obtain the image digest by running the following command:
```

+
[source,bash]
----
oc adm release info <tagged_openshift_release_image> | grep "Pull From"
----
+
Replace `<tagged_openshift_release_image>` with the tagged image for the supported {ocp-short} version. See the following example output:

+
[source,bash]
----
Pull From: quay.io/openshift-release-dev/ocp-release@sha256:69d1292f64a2b67227c5592c1a7d499c7d00376e498634ff8e1946bc9ccdddfe
----

> +

Author: just 28-51 and the line above that.
